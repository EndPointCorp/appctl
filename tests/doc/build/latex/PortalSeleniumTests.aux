\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\select@language{english}
\@writefile{toc}{\select@language{english}}
\@writefile{lof}{\select@language{english}}
\@writefile{lot}{\select@language{english}}
\newlabel{index::doc}{{}{1}{}{section*.2}{}}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}tests/extensions/functional/tests package}{3}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{modules:tests-extensions-functional-tests-package}{{1}{3}{tests/extensions/functional/tests package}{chapter.1}{}}
\newlabel{modules::doc}{{1}{3}{tests/extensions/functional/tests package}{chapter.1}{}}
\newlabel{modules:portal-selenium-tests-documentation}{{1}{3}{tests/extensions/functional/tests package}{chapter.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.1}test\_display\_kiosk module}{3}{section.1.1}}
\newlabel{test_display_kiosk:module-test_display_kiosk}{{1.1}{3}{test\_display\_kiosk module}{section.1.1}{}}
\newlabel{test_display_kiosk::doc}{{1.1}{3}{test\_display\_kiosk module}{section.1.1}{}}
\newlabel{test_display_kiosk:test-display-kiosk-module}{{1.1}{3}{test\_display\_kiosk module}{section.1.1}{}}
\newlabel{test_display_kiosk:test_display_kiosk.TestBaseDisplay}{{1.1}{3}{test\_display\_kiosk module}{section*.3}{}}
\newlabel{test_display_kiosk:test_display_kiosk.TestBaseDisplay.test_widgets_not_displayed}{{1.1}{3}{test\_display\_kiosk module}{section*.4}{}}
\newlabel{test_display_kiosk:test_display_kiosk.TestBaseKioskExtension}{{1.1}{3}{test\_display\_kiosk module}{section*.5}{}}
\newlabel{test_display_kiosk:test_display_kiosk.TestBaseKioskExtension.test_widgets_displayed}{{1.1}{3}{test\_display\_kiosk module}{section*.6}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.2}test\_general module}{4}{section.1.2}}
\newlabel{test_general:test-general-module}{{1.2}{4}{test\_general module}{section.1.2}{}}
\newlabel{test_general::doc}{{1.2}{4}{test\_general module}{section.1.2}{}}
\newlabel{test_general:module-test_general}{{1.2}{4}{test\_general module}{section.1.2}{}}
\newlabel{test_general:test_general.TestMiscellaneous}{{1.2}{4}{test\_general module}{section*.7}{}}
\newlabel{test_general:test_general.TestMiscellaneous.test_eu_cookies_info_bar_is_hidden}{{1.2}{4}{test\_general module}{section*.8}{}}
\newlabel{test_general:test_general.TestSearch}{{1.2}{4}{test\_general module}{section*.9}{}}
\newlabel{test_general:test_general.TestSearch.test_no_searchbox_on_other_planets}{{1.2}{4}{test\_general module}{section*.10}{}}
\newlabel{test_general:test_general.TestSearch.test_search_clicking_search_button}{{1.2}{4}{test\_general module}{section*.11}{}}
\newlabel{test_general:test_general.TestSearch.test_search_hitting_return_on_search_box}{{1.2}{4}{test\_general module}{section*.12}{}}
\newlabel{test_general:test_general.TestSearch.test_search_hitting_return_on_search_button}{{1.2}{4}{test\_general module}{section*.13}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.3}test\_google\_menu module}{4}{section.1.3}}
\newlabel{test_google_menu:test-google-menu-module}{{1.3}{4}{test\_google\_menu module}{section.1.3}{}}
\newlabel{test_google_menu::doc}{{1.3}{4}{test\_google\_menu module}{section.1.3}{}}
\newlabel{test_google_menu:module-test_google_menu}{{1.3}{4}{test\_google\_menu module}{section.1.3}{}}
\newlabel{test_google_menu:test_google_menu.TestGoogleMenu}{{1.3}{4}{test\_google\_menu module}{section*.14}{}}
\newlabel{test_google_menu:test_google_menu.TestGoogleMenu.test_clicking_doodle_item}{{1.3}{4}{test\_google\_menu module}{section*.15}{}}
\newlabel{test_google_menu:test_google_menu.TestGoogleMenu.test_google_items_are_visible_on_click}{{1.3}{4}{test\_google\_menu module}{section*.16}{}}
\newlabel{test_google_menu:test_google_menu.TestGoogleMenu.test_google_menu_is_visible}{{1.3}{5}{test\_google\_menu module}{section*.17}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.4}test\_platform module}{5}{section.1.4}}
\newlabel{test_platform:module-test_platform}{{1.4}{5}{test\_platform module}{section.1.4}{}}
\newlabel{test_platform::doc}{{1.4}{5}{test\_platform module}{section.1.4}{}}
\newlabel{test_platform:test-platform-module}{{1.4}{5}{test\_platform module}{section.1.4}{}}
\newlabel{test_platform:test_platform.TestPlatform}{{1.4}{5}{test\_platform module}{section*.18}{}}
\newlabel{test_platform:test_platform.TestPlatform.test_get_chrome_gpu_data}{{1.4}{5}{test\_platform module}{section*.19}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.5}test\_ros module}{5}{section.1.5}}
\newlabel{test_ros:module-test_ros}{{1.5}{5}{test\_ros module}{section.1.5}{}}
\newlabel{test_ros::doc}{{1.5}{5}{test\_ros module}{section.1.5}{}}
\newlabel{test_ros:test-ros-module}{{1.5}{5}{test\_ros module}{section.1.5}{}}
\newlabel{test_ros:test_ros.TestBaseSingleBrowserROS}{{1.5}{5}{test\_ros module}{section*.20}{}}
\newlabel{test_ros:test_ros.TestBaseSingleBrowserROS.test_ros_position_after_search}{{1.5}{5}{test\_ros module}{section*.21}{}}
\newlabel{test_ros:test_ros.TestBaseTwoBrowsersROS}{{1.5}{5}{test\_ros module}{section*.22}{}}
\newlabel{test_ros:test_ros.TestBaseTwoBrowsersROS.test_ros_positions_in_browsers_aligned_after_kiosk_search}{{1.5}{5}{test\_ros module}{section*.23}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.6}test\_runway module}{6}{section.1.6}}
\newlabel{test_runway:test-runway-module}{{1.6}{6}{test\_runway module}{section.1.6}{}}
\newlabel{test_runway:module-test_runway}{{1.6}{6}{test\_runway module}{section.1.6}{}}
\newlabel{test_runway::doc}{{1.6}{6}{test\_runway module}{section.1.6}{}}
\newlabel{test_runway:test_runway.TestRunway}{{1.6}{6}{test\_runway module}{section*.24}{}}
\newlabel{test_runway:test_runway.TestRunway.prepare_poi}{{1.6}{6}{test\_runway module}{section*.25}{}}
\newlabel{test_runway:test_runway.TestRunway.test_runway_buttons_basic}{{1.6}{6}{test\_runway module}{section*.26}{}}
\newlabel{test_runway:test_runway.TestRunway.test_runway_check_earth_icon_click}{{1.6}{6}{test\_runway module}{section*.27}{}}
\newlabel{test_runway:test_runway.TestRunway.test_runway_planets_on_max_zoom_out}{{1.6}{6}{test\_runway module}{section*.28}{}}
\newlabel{test_runway:test_runway.TestRunway.test_runway_points_of_interest}{{1.6}{6}{test\_runway module}{section*.29}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.7}test\_zoom module}{6}{section.1.7}}
\newlabel{test_zoom::doc}{{1.7}{6}{test\_zoom module}{section.1.7}{}}
\newlabel{test_zoom:module-test_zoom}{{1.7}{6}{test\_zoom module}{section.1.7}{}}
\newlabel{test_zoom:test-zoom-module}{{1.7}{6}{test\_zoom module}{section.1.7}{}}
\newlabel{test_zoom:test_zoom.TestZoomButtons}{{1.7}{6}{test\_zoom module}{section*.30}{}}
\newlabel{test_zoom:test_zoom.TestZoomButtons.test_zoom_buttons}{{1.7}{6}{test\_zoom module}{section*.31}{}}
\newlabel{test_zoom:test_zoom.TestZoomButtons.test_zoom_in_button_change}{{1.7}{6}{test\_zoom module}{section*.32}{}}
\newlabel{test_zoom:test_zoom.TestZoomButtons.test_zoom_out_button_change}{{1.7}{6}{test\_zoom module}{section*.33}{}}
\@writefile{toc}{\contentsline {chapter}{Python Module Index}{7}{section*.34}}
\@writefile{toc}{\contentsline {chapter}{Index}{9}{section*.35}}
