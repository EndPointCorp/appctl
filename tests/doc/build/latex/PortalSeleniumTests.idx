\indexentry{test\_display\_kiosk (module)|hyperpage}{3}
\indexentry{TestBaseDisplay (class in test\_display\_kiosk)|hyperpage}{3}
\indexentry{test\_widgets\_not\_displayed() (test\_display\_kiosk.TestBaseDisplay method)|hyperpage}{3}
\indexentry{TestBaseKioskExtension (class in test\_display\_kiosk)|hyperpage}{3}
\indexentry{test\_widgets\_displayed() (test\_display\_kiosk.TestBaseKioskExtension method)|hyperpage}{3}
\indexentry{test\_general (module)|hyperpage}{4}
\indexentry{TestMiscellaneous (class in test\_general)|hyperpage}{4}
\indexentry{test\_eu\_cookies\_info\_bar\_is\_hidden() (test\_general.TestMiscellaneous method)|hyperpage}{4}
\indexentry{TestSearch (class in test\_general)|hyperpage}{4}
\indexentry{test\_no\_searchbox\_on\_other\_planets() (test\_general.TestSearch method)|hyperpage}{4}
\indexentry{test\_search\_clicking\_search\_button() (test\_general.TestSearch method)|hyperpage}{4}
\indexentry{test\_search\_hitting\_return\_on\_search\_box() (test\_general.TestSearch method)|hyperpage}{4}
\indexentry{test\_search\_hitting\_return\_on\_search\_button() (test\_general.TestSearch method)|hyperpage}{4}
\indexentry{test\_google\_menu (module)|hyperpage}{4}
\indexentry{TestGoogleMenu (class in test\_google\_menu)|hyperpage}{4}
\indexentry{test\_clicking\_doodle\_item() (test\_google\_menu.TestGoogleMenu method)|hyperpage}{4}
\indexentry{test\_google\_items\_are\_visible\_on\_click() (test\_google\_menu.TestGoogleMenu method)|hyperpage}{4}
\indexentry{test\_google\_menu\_is\_visible() (test\_google\_menu.TestGoogleMenu method)|hyperpage}{5}
\indexentry{test\_platform (module)|hyperpage}{5}
\indexentry{TestPlatform (class in test\_platform)|hyperpage}{5}
\indexentry{test\_get\_chrome\_gpu\_data() (test\_platform.TestPlatform method)|hyperpage}{5}
\indexentry{test\_ros (module)|hyperpage}{5}
\indexentry{TestBaseSingleBrowserROS (class in test\_ros)|hyperpage}{5}
\indexentry{test\_ros\_position\_after\_search() (test\_ros.TestBaseSingleBrowserROS method)|hyperpage}{5}
\indexentry{TestBaseTwoBrowsersROS (class in test\_ros)|hyperpage}{5}
\indexentry{test\_ros\_positions\_in\_browsers\_aligned\_after\_kiosk\_search() (test\_ros.TestBaseTwoBrowsersROS method)|hyperpage}{5}
\indexentry{test\_runway (module)|hyperpage}{6}
\indexentry{TestRunway (class in test\_runway)|hyperpage}{6}
\indexentry{prepare\_poi() (test\_runway.TestRunway method)|hyperpage}{6}
\indexentry{test\_runway\_buttons\_basic() (test\_runway.TestRunway method)|hyperpage}{6}
\indexentry{test\_runway\_check\_earth\_icon\_click() (test\_runway.TestRunway method)|hyperpage}{6}
\indexentry{test\_runway\_planets\_on\_max\_zoom\_out() (test\_runway.TestRunway method)|hyperpage}{6}
\indexentry{test\_runway\_points\_of\_interest() (test\_runway.TestRunway method)|hyperpage}{6}
\indexentry{test\_zoom (module)|hyperpage}{6}
\indexentry{TestZoomButtons (class in test\_zoom)|hyperpage}{6}
\indexentry{test\_zoom\_buttons() (test\_zoom.TestZoomButtons method)|hyperpage}{6}
\indexentry{test\_zoom\_in\_button\_change() (test\_zoom.TestZoomButtons method)|hyperpage}{6}
\indexentry{test\_zoom\_out\_button\_change() (test\_zoom.TestZoomButtons method)|hyperpage}{6}
