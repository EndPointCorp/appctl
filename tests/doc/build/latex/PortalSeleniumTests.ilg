This is makeindex, version 2.15 [TeX Live 2013] (kpathsea + Thai support).
Scanning style file ./python.ist......done (6 attributes redefined, 0 ignored).
Scanning input file PortalSeleniumTests.idx....done (38 entries accepted, 0 rejected).
Sorting entries....done (207 comparisons).
Generating output file PortalSeleniumTests.ind....done (55 lines written, 0 warnings).
Output written in PortalSeleniumTests.ind.
Transcript written in PortalSeleniumTests.ilg.
