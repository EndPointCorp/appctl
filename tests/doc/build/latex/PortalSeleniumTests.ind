\begin{theindex}
\def\bigletter#1{{\Large\sffamily#1}\nopagebreak\vspace{1mm}}

  \bigletter P
  \item prepare\_poi() (test\_runway.TestRunway method), \hyperpage{6}

  \indexspace
  \bigletter T
  \item test\_clicking\_doodle\_item() (test\_google\_menu.TestGoogleMenu method), \hyperpage{4}
  \item test\_display\_kiosk (module), \hyperpage{3}
  \item test\_eu\_cookies\_info\_bar\_is\_hidden() (test\_general.TestMiscellaneous method), 
		\hyperpage{4}
  \item test\_general (module), \hyperpage{4}
  \item test\_get\_chrome\_gpu\_data() (test\_platform.TestPlatform method), \hyperpage{5}
  \item test\_google\_items\_are\_visible\_on\_click() (test\_google\_menu.TestGoogleMenu method), 
		\hyperpage{4}
  \item test\_google\_menu (module), \hyperpage{4}
  \item test\_google\_menu\_is\_visible() (test\_google\_menu.TestGoogleMenu method), \hyperpage{5}
  \item test\_no\_searchbox\_on\_other\_planets() (test\_general.TestSearch method), \hyperpage{4}
  \item test\_platform (module), \hyperpage{5}
  \item test\_ros (module), \hyperpage{5}
  \item test\_ros\_position\_after\_search() (test\_ros.TestBaseSingleBrowserROS method), 
		\hyperpage{5}
  \item test\_ros\_positions\_in\_browsers\_aligned\_after\_kiosk\_search() (test\_ros.TestBaseTwoBrowsersROS method), 
		\hyperpage{5}
  \item test\_runway (module), \hyperpage{6}
  \item test\_runway\_buttons\_basic() (test\_runway.TestRunway method), \hyperpage{6}
  \item test\_runway\_check\_earth\_icon\_click() (test\_runway.TestRunway method), \hyperpage{6}
  \item test\_runway\_planets\_on\_max\_zoom\_out() (test\_runway.TestRunway method), \hyperpage{6}
  \item test\_runway\_points\_of\_interest() (test\_runway.TestRunway method), \hyperpage{6}
  \item test\_search\_clicking\_search\_button() (test\_general.TestSearch method), \hyperpage{4}
  \item test\_search\_hitting\_return\_on\_search\_box() (test\_general.TestSearch method), 
		\hyperpage{4}
  \item test\_search\_hitting\_return\_on\_search\_button() (test\_general.TestSearch method), 
		\hyperpage{4}
  \item test\_widgets\_displayed() (test\_display\_kiosk.TestBaseKioskExtension method), 
		\hyperpage{3}
  \item test\_widgets\_not\_displayed() (test\_display\_kiosk.TestBaseDisplay method), 
		\hyperpage{3}
  \item test\_zoom (module), \hyperpage{6}
  \item test\_zoom\_buttons() (test\_zoom.TestZoomButtons method), \hyperpage{6}
  \item test\_zoom\_in\_button\_change() (test\_zoom.TestZoomButtons method), \hyperpage{6}
  \item test\_zoom\_out\_button\_change() (test\_zoom.TestZoomButtons method), \hyperpage{6}
  \item TestBaseDisplay (class in test\_display\_kiosk), \hyperpage{3}
  \item TestBaseKioskExtension (class in test\_display\_kiosk), \hyperpage{3}
  \item TestBaseSingleBrowserROS (class in test\_ros), \hyperpage{5}
  \item TestBaseTwoBrowsersROS (class in test\_ros), \hyperpage{5}
  \item TestGoogleMenu (class in test\_google\_menu), \hyperpage{4}
  \item TestMiscellaneous (class in test\_general), \hyperpage{4}
  \item TestPlatform (class in test\_platform), \hyperpage{5}
  \item TestRunway (class in test\_runway), \hyperpage{6}
  \item TestSearch (class in test\_general), \hyperpage{4}
  \item TestZoomButtons (class in test\_zoom), \hyperpage{6}

\end{theindex}
