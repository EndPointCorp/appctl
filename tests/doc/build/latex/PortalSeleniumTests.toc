\select@language {english}
\contentsline {chapter}{\numberline {1}tests/extensions/functional/tests package}{3}{chapter.1}
\contentsline {section}{\numberline {1.1}test\_display\_kiosk module}{3}{section.1.1}
\contentsline {section}{\numberline {1.2}test\_general module}{4}{section.1.2}
\contentsline {section}{\numberline {1.3}test\_google\_menu module}{4}{section.1.3}
\contentsline {section}{\numberline {1.4}test\_platform module}{5}{section.1.4}
\contentsline {section}{\numberline {1.5}test\_ros module}{5}{section.1.5}
\contentsline {section}{\numberline {1.6}test\_runway module}{6}{section.1.6}
\contentsline {section}{\numberline {1.7}test\_zoom module}{6}{section.1.7}
\contentsline {chapter}{Python Module Index}{7}{section*.34}
\contentsline {chapter}{Index}{9}{section*.35}
