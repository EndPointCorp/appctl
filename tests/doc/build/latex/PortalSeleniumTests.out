\BOOKMARK [0][-]{chapter.1}{tests/extensions/functional/tests package}{}% 1
\BOOKMARK [1][-]{section.1.1}{test\137display\137kiosk module}{chapter.1}% 2
\BOOKMARK [1][-]{section.1.2}{test\137general module}{chapter.1}% 3
\BOOKMARK [1][-]{section.1.3}{test\137google\137menu module}{chapter.1}% 4
\BOOKMARK [1][-]{section.1.4}{test\137platform module}{chapter.1}% 5
\BOOKMARK [1][-]{section.1.5}{test\137ros module}{chapter.1}% 6
\BOOKMARK [1][-]{section.1.6}{test\137runway module}{chapter.1}% 7
\BOOKMARK [1][-]{section.1.7}{test\137zoom module}{chapter.1}% 8
\BOOKMARK [0][-]{section*.34}{Python Module Index}{}% 9
\BOOKMARK [0][-]{section*.35}{Index}{}% 10
